ThreeDSecure/src/test/java/com/braintreepayments/api/ThreeDSecureClientUnitTest.java

```diff
@@ -245,6 +245,12 @@ public void onActivityResult_whenResultNotOk_doesNothing() {
         verifyNoMoreInteractions(braintreeClient);
         sut.onActivityResult(AppCompatActivity.RESULT_CANCELED, new Intent(), threeDSecureResultCallback);
         verifyNoMoreInteractions(braintreeClient);
+
+        ArgumentCaptor<BraintreeException> captor = ArgumentCaptor.forClass(BraintreeException.class);
+        verify(threeDSecureResultCallback).onResult((ThreeDSecureResult) isNull(), captor.capture());
+
+        BraintreeException exception = captor.getValue();
+        assertEquals("User canceled 3DS.", exception.getMessage());
```

Comment:
```suggestion
        Exception exception = captor.getValue();
        assertTrue(exception instanceof BraintreeException);
        assertEquals("User canceled 3DS.", exception.getMessage());
```
Nitpick - we could update these tests to all also assert on the Exception type.